sdk/scripts/check-crates.sh

```diff
@@ -25,6 +25,9 @@ printf "%s\n" "${files[@]}"
 
 has_error=0
 for file in "${files[@]}"; do
+  if [ -z $file ]; then
```

Comment:
shellcheck is complaining about this
```suggestion
  if [ -z "$file" ]; then
```